aries_cloudagent/config/argparse.py

```diff
@@ -352,8 +352,6 @@ def get_settings(self, args: Namespace) -> dict:
             settings["debug.invite_multi_use"] = True
         if args.invite_public:
             settings["debug.invite_public"] = True
-        if args.test_suite_endpoint:
-            settings["debug.test_suite_endpoint"] = args.test_suite_endpoint
```

Comment:
Is this removed intentionally?

Reply:
Oops - I think I should put it back, and see about the effects on the test suite's static connections. Don't approve the PR until I do - probably late this (my) afteroon.